README.md

```diff
@@ -1,3 +1,144 @@
-# Wix build rules and macros for Bazel
+<h1>Wix build rules and macros for Bazel</h1>
+
+<h3 id="wix-logo">
+	<img src="assets/icons/bazel.svg" height="80" width="80">BUILD 
+	<img src="assets/icons/wix.svg" height="80" width="140"> EXTEND
+</h3>
+
+<br>
+
+A place for rules and macros that were built inside [Wix](https://www.wix.engineering/).
+
+[![Build status](https://github.com/wix-playground/wix_build_tools/actions/workflows/continuous-integration-workflow.yml/badge.svg)](https://github.com/wix-playground/wix_build_tools/actions/workflows/continuous-integration-workflow.yml)
+
+<br>
+
+- [Overview](#overview)
+- [Getting Started](#getting-started)
+- [Rules](#rules)
+   - [git_cached_repository](#git_cached_repository)
+      - [Usage](#git_cached_repository_usage)
+      - [Attributes](#git_cached_repository_attr)
+- [Contributing Guidelines](#contributing)
+- [FAQ](#faq)
+
+<br>
+
+<h2 id="overview">Overview</h2>
+
+This repository contains a set of rules created and used within Wix.
+It was created to solve requirements which did not have an off-the-shelf solutions to cope with challenges at large scale.
+Please note that some offering of this repository might be in an alpha/beta phase, be aware before adopting solutions based on this repository.
```

Comment:
Let's be honest - this repo does not have "set of rules" but one rule :) 
"Used within Wix" is problematic w.r.t allowing others from outside to feel welcomed to introduce changes / issues / ideas :) 

```suggestion
This repository currently contains the `git_cached_repository` rule. Please note that this is still in alpha phase. Feel free to try it out but be aware that there may still be uncovered cases, and that the API might change.
```

Reply:
I'm optimistic, we have a lot more to offer that can can be integrated into this repository.
Need to remember to align this section once additional rules/macros would get added.